README.md

```diff
@@ -593,7 +593,7 @@ Generates the following response payload:
 
 All 500 errors hide your message from the end user. Your message is recorded in the server log.
 
-### `Boom.badImplementation([message], [data])`
+### `Boom.badImplementation([message], [data])` - alias: `internal`
```

Comment:
Can you change this to 
`Boom.badImplementation([message], [data])` - (*alias: `internal`*)